src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk;
+
+import java.io.IOException;
+
+import org.apache.http.HttpHost;
+
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.client.RestClient;
+import org.opensearch.client.RestClientBuilder;
+import org.opensearch.client.json.jackson.JacksonJsonpMapper;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.transport.OpenSearchTransport;
+import org.opensearch.client.transport.rest_client.RestClientTransport;
+
+/**
+ * This class creates a Client for SDK to make requests to OpenSearch
+ */
+public class SDKClient {
+    private final Logger logger = LogManager.getLogger(SDKClient.class);
+    private OpenSearchClient client;
+    private RestClient restClient = null;
+
+    /**
+     * Creates client for SDK
+     * @param hostAddress The address client can connect to
+     * @param port The port of the address
+     * @throws IOException if client failed
+     * @return SDKClient
+     */
+    public OpenSearchClient createClient(String hostAddress, int port) throws IOException {
+        RestClientBuilder builder = RestClient.builder(new HttpHost(hostAddress, port));
+        builder.setStrictDeprecationMode(true);
```

Comment:
Curious, what does this do?
Side note: I could only find elasticsearch documentation[1]. 
@VachaShah 

[1] https://javadoc.io/static/org.elasticsearch.client/elasticsearch-rest-client/6.8.3/org/elasticsearch/client/RestClientBuilder.html